DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateGroupsController.cs

```diff
@@ -47,6 +59,51 @@ public IActionResult GroupDelegates(int groupId, int page = 1)
             return View(model);
         }
 
+        [HttpGet]
+        [Route("{groupId:int}/Delegates/Remove/{delegateId:int}")]
+        public IActionResult GroupDelegatesRemove(int groupId, int delegateId)
+        {
+            var centreId = User.GetCentreId();
+            var groupName = groupsDataService.GetGroupName(groupId, centreId);
+            var delegateUser = userDataService.GetDelegateUserById(delegateId);

```

Comment:
Do we need this call if we're fetching groupDelegates (could we grab the delegate off groupdelegates or is that missing some data)?